master/internal/core.go

```diff
@@ -306,17 +306,27 @@ func (m *Master) getAggregatedResourceAllocation(c echo.Context) error {
 	}
 
 	for _, entry := range resp.ResourceEntries {
-		for label, seconds := range entry.ByLabel {
-			if err = write("label", label, entry.PeriodStart, seconds); err != nil {
-				return err
+		do := func(agg string, vals map[string]float32) error {
```

Comment:
Is there a more descriptive name we could use than `do`?

Reply:
Perhaps I took the names-in-smaller-scopes-should-be-shorter principle too far. :slightly_smiling_face:  Renaming to `writeAggType`.